### PR TITLE
Mount /var/lib/kubelet in system containers

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

**Comment by vrutkovs** [comment]
cc @giuseppe

File: roles/openshift_node/tasks/node_system_container.yml [hunk 1]
```diff
@@ -12,6 +12,13 @@
     path: "/etc/origin/openvswitch"
     mode: '0750'
 
+# TODO: remove when system container is fixed to not include it
```
Comment (giuseppe) [inline roles/openshift_node/tasks/node_system_container.yml]: should we change the system container to create this directory or is it fine to ensure it exists from the installer?  If the system container needs write access to /var/lib/kubelet then it should exist somewhere, so I think it is fine to leave this part anyway.

File: roles/openshift_node/defaults/main.yml [hunk 2]
```diff
@@ -100,6 +100,13 @@ r_openshift_node_use_firewalld: "{{ os_firewall_use_firewalld | default(False) }
 
 l_is_node_system_container: "{{ openshift_use_node_system_container | default(openshift_use_system_containers | default(openshift_is_containerized | default(false))) | bool }}"
 
+openshift_kubelet_mounts_l:
+- type: bind
+  source: "/var/lib/kubelet"
+  destination: "/var/lib/kubelet"
+  options:
+  - bind
+
```
Comment (giuseppe) [inline roles/openshift_node/defaults/main.yml]: if newer versions of Origin always require this directory, then I think it is better to move it in the system container itself.

**Comment by giuseppe** [comment]
@smarterclayton the system containers tests pass with https://github.com/openshift/openshift-ansible/pull/7872.  Since this PR is basically doing the same thing, tests should be fine once it is merged.

**Comment by vrutkovs** [comment]
@giuseppe okay, closing this one then. I've made sure no other fixes are required to unblock the testing

**Pull request closed without merging**